src/game_server/player_update_packet.rs

```diff
@@ -21,6 +36,215 @@ impl SerializePacket for PlayerUpdateOpCode {
     }
 }
 
+#[derive(SerializePacket, DeserializePacket)]
+pub struct LootEvent {
+	guid: u64,
+	position: Pos,
+	rotation: Pos,
+	model_name: String,
+}
+
+impl GamePacket for LootEvent {
+	type Header = PlayerUpdateOpCode;
+	const HEADER: Self::Header = PlayerUpdateOpCode::LootEvent;
+}
+
+#[derive(SerializePacket, DeserializePacket)]
+pub struct HudMessage {
+	guid: u64,
+	unkguid: u64,
```

Comment:
```suggestion
	unknown_guid: u64,
```
nit: snake case to conform to [Rust conventions](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html)